io.github.Fontforge.appdata.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Joe Da Silva <digital@joescat.com> -->
+<component type="desktop">
+  <id>io.github.Fontforge</id>
+  <launchable type="desktop-id">io.github.Fontforge.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <description>
+    <p xml:lang="C">
+  FontForge is a font editor for outline and bitmap fonts that lets you create,
+  edit, or convert, a range of fonts, including PostScript, TrueType, OpenType,
+  cid-keyed, multi-master, cff, SVG and BitMap (bdf, FON, NFNT) fonts.</p>
+    <p xml:lang="C">
+  FontForge is free Open Source Software written to run on various computer operating
+  systems. You can use FontForge Graphically or as a command-line tool.</p>
+    <p xml:lang="C">
+  Learning to use FontForge is easy, and there are various tutorials available beginning
+  with the basics up to more advanced features such as making and using scripts.</p>
+    <p xml:lang="es">
```

Comment:
If they have any reasonable build system and uses gettext then it should handle translations itself.

Reply:
This is from their file, i.e. they have this right now. It's not my modification.

Reply:
FWIW: I am proposing a new appdata file here: https://github.com/fontforge/fontforge/pull/3362 feel free to comment there.